cassandra/src/main/scala/org/openzipkin/dependencies/spark/cassandra/DependenciesInfo.scala

```diff
@@ -0,0 +1,44 @@
+package org.openzipkin.dependencies.spark.cassandra
+
+import com.twitter.zipkin.common.{DependencyLink, Dependencies}
+
+/*
+ * A workaround for https://github.com/openzipkin/zipkin/issues/867
+ * The DependencyLink is not serializable so we must wrap it here.
```

Comment:
I think it's better this way anyway, less dependencies on the merging code elsewhere.